plugins/modules/network_compliance_workflow_manager.py

```diff
@@ -613,7 +626,7 @@ def get_device_ids_from_site(self, site_name, site_id):
         Retrieves the management IP addresses and their corresponding instance UUIDs of devices associated with a specific site in Cisco Catalyst Center.
 
         Parameters:
-            site_name (str): The name of the site whose devices" information is to be retrieved.
+            site_name (str): The name of the site whose devices' information is to be retrieved.
```

Comment:
Many modules using get_device_ids_from_site.. Can we put into dnac.py under sites section?

./swim_workflow_manager.py
./site_workflow_manager.py
./site_intent.py
./network_compliance_workflow_manager.py
./accesspoint_workflow_manager.py
./device_configs_backup_workflow_manager.py
./swim_intent.py
./provision_workflow_manager.py

Reply:
This function added inside the dnac.py